.buildkite/deploy/kubernetes/scripts/setup-k8s-env.sh

```diff
@@ -4,11 +4,15 @@ set -euo pipefail
 
 source .buildkite/env-scripts/util.sh
 
+export KUBECONFIG="${WORKSPACE}/kubecfg"
+export BIN="${WORKSPACE}/bin"
+add_bin_path
+
 echo "--- Installing kind & kubectl"
-retry_with_count 5 .buildkite/deploy/kubernetes/scripts/install-kind.sh
-retry_with_count 5 .buildkite/deploy/kubernetes/scripts/install-kubectl.sh
+asdf plugin add kind
+asdf install kind $ASDF_KIND_VERSION
 
-echo "--- Setting up kind"
+echo "~~~ Setting up kind"
```

Comment:
I suggest we add `echo "~~~ Continuing steps"` (or something similar) at the bottom of this script, otherwise, the actual test execution output gets folded under the section "Setting up kind" as we can see in: 
https://buildkite.com/elastic/deploy-k8s/builds/4490#018f5718-08bc-4960-8667-35385b6102f5/165